### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by stblender** [markbegma/homebridge-hdl-buspro#4] [General]
Hey! Just wanted to say thank you for this, it's been a great help as my house is using HDL and this has made the whole home automation project worth it

**Comment by markbegma** [comment]
> Hey! Just wanted to say thank you for this, it's been a great help as my house is using HDL and this has made the whole home automation project worth it

You are welcome, @stblender :) it is nice to hear that a half-baked personal project can help others.

If not a secret, what kind of devices do you automate with the package, what Apple gadget do you control home with and how does it perform? I barely get feedback from anyone.

**Reply by stblender** [reply]
Sure, happy to provide feedback.. Well in terms of the HDL products, I have mostly relay lights on/off 12ch & 4ch, a few dimmers and a curtain controller, in total it adds up to around 50+ channels/accessories that end up in homebridge. Within the house I have the touch panels to control everything.

Apple devices: a few iphones, ipad, apple tv (as a hub) and macbook..

For the relay lights, they work perfectly, no delays at all in local network, the only delays have been due to wifi/data connectivity. I especially like the speed in which the physical touch panels and homekit would sync the on/off status - it makes the whole thing work like a native homekit device. I've purchased some official homekit devices and they have not performed as well as this to be honest.

The relay dimmers are similar, but the lights seem to flash to 100% whenever you are moving the dimmer up or down. It works, but I am not sure if there's a setting or tweak I can do to make it smoother - I haven't tweaked around too much tbh.

The curtains do work, but not as well as the lights. It may have to do with how it's setup/integrated between the HDL control and Somfy side as I don't actually have control over the 'percentage' of opening/closing - it's just open or close and it goes all the way each time. I plan on investigating/tweaking with it at some point, but it's a minor issue tbh. However, homekit/homebridge don't get updated with the status (unlike the lights), so if I open the curtains using the physical touch panels, homekit still thinks they are closed, and so it does limit potential automations.

**Comment by markbegma** [comment]
Hi @stblender ,

Might be kind of overdue, but I had time to return to this package recently and might have fixed your issue with dimmers as well as status syncs in the newest update. So you might want to check it out ;)

**Comment by stblender** [comment]
Hey that’s awesome news, thanks, I will test it out.
